apps/intellij-plugin/build.gradle.kts

```
@@ -0,0 +1,59 @@
+import org.gradle.api.JavaVersion.VERSION_17
+
+plugins {
+    id("java")
+    id("org.jetbrains.kotlin.jvm") version "1.7.20"
+    id("org.jetbrains.intellij") version "1.12.0"
+}
+
+group = "ai.hash"
+version = "1.0-SNAPSHOT"
```

Where this version is used? Shouldn't it be something like `0.0.1`?

---

on the JetBrains Marketplace, I can switch it to `0.0.1`, `1.0-SNAPSHOT` was simply the default from the template I generated the plugin from.

---

Yes, let's switch it 🙂 
```suggestion
version = "0.0.1"
```